packages/yocaml/yocaml.2.0.1/opam

```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+synopsis: "Core engine of the YOCaml Static Site Generator"
+description: "YOCaml is a build system dedicated to generate static document"
+maintainer: ["The XHTMLBoy <xhtmlboi@gmail.com>"]
+authors: ["The XHTMLBoy <xhtmlboi@gmail.com>"]
+license: "GPL-3.0-or-later"
+homepage: "https://github.com/xhtmlboi/yocaml"
+bug-reports: "https://github.com/xhtmlboi/yocaml/issues"
+depends: [
+  "ocaml" {>= "5.1.1" & < "5.3"}
+  "dune" {>= "3.14" & >= "3.0.0"}
```

Comment:
```suggestion
  "dune" {>= "3.14"}
```
To avoid this, please remove dune from the dependencies in the dune-project file, dune adds itself automatically to the generated opam file